src/main/webapp/js/login.js

```diff
@@ -0,0 +1,41 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+function onSignIn(googleUser) {
```

Comment:
Actually these functions need comments. Also this file should be in `/js/pages/` right?

Reply:
I wasn't sure whether to put it in js/pages because it's not tied to a specific page, but I'll move it in

Reply:
It has an event listener in it so I just assumed it was a page. But put it where you feel like it should go